src/app.js

```diff
@@ -481,9 +493,13 @@
 			}
 		});
 		
-		notifyAll('app.init', {
-			page: currentPage
-		});
+		if (!currentPage) {
+			App.log({args: 'Can not init application: No current page set.', fx: 'error'});
+		} else {
+			notifyAll('app.init', {
+				page: currentPage
+			});
+		}
```

Comment:
Should we still call `notifyAll('app.init')` even if the page was not found ? module should still work, no ?

Reply:
go with already notify

Reply:
But Some module may need fix and may crash with this new edge case. They actually rely on the existence of a page not null.

Reply:
Should we give them an empty one ? (null object pattern) ?

Reply:
But remember that we always called app.init, even if the page was not found.

Reply:
Ok maybe we should create a new handler for this case ?